app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable,
+         :confirmable
```

Comment:
Hey team, when I try to register an account, this line ensures that a user verifies their account from a message sent to their mail. However, I do not get such mail. In order to get this issue resolved, you can utilize gems like letter-opener for this so that a user can be verified.

Reply:
@duane, the email is sent however, in development mode, the mail is only sent to the server terminal. Kindly verify that and click on the link to verify.

Reply:
Gem implemented. Thank you for the suggestion. 👍🏾